engines/sci/graphics/font.cpp

```diff
@@ -38,7 +38,9 @@ GfxFontFromResource::GfxFontFromResource(ResourceManager *resMan, GfxScreen *scr
 
 	_resource = resMan->findResource(ResourceId(kResourceTypeFont, resourceId), true);
 	if (!_resource) {
-		error("font resource %d not found", resourceId);
+		// The Lighthouse demo does not contain any fonts
```

Comment:
Just tested the Lighthouse demo, and it works without a loaded font (commented out the getFont call).
Allowing a getFont(0) to fail definitely isn't right, but allowing getFont(65535) to fail might be, assuming Colin is right that nothing else requires it at the moment. That means changing this variable initialization
`int16 GfxText32::_defaultFontId = 0;`
to read 65535 (or -1) instead and special-case *that* (I'm a bit worried that this variable is signed; should I be?). If it turns out that something else does need this, of course, supporting an actual embedded font would be the answer.